astropy/io/fits/tests/test_convenience.py

```diff
@@ -61,3 +60,36 @@ def test_table_to_hdu(self, tmpdir):
         assert isinstance(hdu, fits.BinTableHDU)
         filename = str(tmpdir.join('test_table_to_hdu.fits'))
         hdu.writeto(filename, overwrite=True)
+
+    def test_printdiff(self):
+        """
+        Test that FITSDiff can run the different inputs without crashing.
+        """
+
+        # Testing different string input options
+        assert printdiff(self.data('arange.fits'), self.data('blank.fits')) is None
+        assert printdiff(self.data('arange.fits'), self.data('blank.fits'), ext=0) is None
+        assert printdiff(self.data('o4sp040b0_raw.fits'), self.data('o4sp040b0_raw.fits'), extname='sci') is None
```

Comment:
Yeah... so I thought my pycharm line indicator was at 80, when it was actually at 120.  I have now switched the defaults and am going to fix all the files i've edited...